action.yml

```diff
@@ -21,9 +22,10 @@ inputs:
     description: 'Extra command-line arguments to pass to cachix. If empty, defaults to -j8'
   skipAddingSubstituter:
     description: 'Set to true to skip adding cachix cache as a substitute'
+    default: 'false'
   useDaemon:
     description: "Push store paths to the cache as they're built with the Cachix Daemon"
-    default: true
+    default: 'true'
```

Comment:
According to the GH Actions documentation, the default value of an input should be specified as a string:

<img width="736" alt="image" src="https://github.com/cachix/cachix-action/assets/2049686/4aef9f4d-d64a-41af-a53b-9ba18e1cd62f">


Reference: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault